### PR TITLE
Dao worlds changes

### PR DESCRIPTION
**Pull request by angelol**
This contains all the fixes I've made to make it work for alienworlds as well as the table and action renames.

### CODE REVIEW

File: src/components/ui/msig-proposal.vue [hunk 1]
```diff
@@ -523,89 +475,13 @@ export default {
         this.transactionCallback('e_unapproval')
       }
     },
-    // deny a proposal, just marks as an action
-    async denyProposal (proposer, proposalName) {
```
Comment (dallasjohnson) [inline src/components/ui/msig-proposal.vue:527]: Seeing that you removed this makes me think we might need a `denyproposal` action on the msig contract. it doesn't necessarily need to have a logical impact on the state of a msig but it should an existing approval if there is one. I remember we added the denyproposal action to signal that custodians had seen the proposal and deliberately chosen to deny - which is different than simply ignoring a prop.

File: src/pages/manage-candidateship.vue [hunk 2]
```diff
@@ -363,7 +363,7 @@ export default {
 
       let registeraction = {
         account: this.$dir.getAccount(this.$dir.ACCOUNT_CUSTODIAN),
-        name: 'nominatecane',
+        name: 'nominatecand',
```
Comment (dallasjohnson) [inline src/pages/manage-candidateship.vue:366]: Shouldn't this be `nominatecane` now?

File: src/pages/manage-candidateship.vue [hunk 3]
```diff
@@ -392,7 +392,7 @@ export default {
       let actions = [
         {
           account: this.$dir.getAccount(this.$dir.ACCOUNT_CUSTODIAN),
-          name: 'withdrawcane',
+          name: 'withdrawcand',
```
Comment (dallasjohnson) [inline src/pages/manage-candidateship.vue:395]: and this one `withdrawcane`

**Review by dallasjohnson — CHANGES_REQUESTED** [review]
I think a couple of the actions were wrong so might need changing.

File: src/pages/manage-candidateship.vue (hunk 2 above)
Comment (angelol) [reply src/pages/manage-candidateship.vue]: Sorry, this got somehow lost in merging, I should have double-checked

File: src/pages/manage-candidateship.vue (hunk 3 above)
Comment (angelol) [reply src/pages/manage-candidateship.vue]: This too

**Review by dallasjohnson — APPROVED** [review]
Thank you 👍

File: src/components/ui/msig-proposal.vue (hunk 1 above)
Comment (angelol) [reply src/components/ui/msig-proposal.vue:527]: Okay, I'll add a separate task for this. The buttons were commented out in the UI, so it looked like it's no longer in use. I suggest we merge this as is and have this change in a separate branch.

**Pull request merged**